plugin-server/src/cdp/services/hog-executor.service.test.ts

```diff
@@ -164,6 +165,41 @@ describe('Hog Executor', () => {
             expect(result.error).toBeUndefined()
         })
 
+        it('can handle selecting entire object', () => {
```

Comment:
Can we add another test to check "allows selecting entire object with overrides". So we can do 

```
                        value: {
                            [EXTEND_OBJECT_KEY]: '{person.properties}',
			   'email': "REDACTED"
                        },
```

The current implementation doesn't allow this but I think we should definitely make it work as its a cool feature